Source/Core/Core/MemoryWatcher.cpp

```diff
@@ -107,6 +146,12 @@ void MemoryWatcher::WatcherThread()
 					0,
 					reinterpret_cast<sockaddr*>(&m_addr),
 					sizeof(m_addr));
+#elif defined(_WIN32)
+				if(!WriteFile(pipe, message, sizeof(char) * strlen(message), NULL, NULL))
```

Comment:
This line alone tells me you haven't tested this code. Passing a std::string to strlen isn't going to work.